trace/trace.go

```diff
@@ -110,6 +111,9 @@ const (
 	SpanKindClient
 )
 
+// DefaultBufferLimit is the default value for trace.StartOptions.BufferLimit.
+const DefaultBufferLimit = 1000
```

Comment:
I think we need to be consistent and allow user to override this default via trace.ApplyConfig. It is not represented in the specs and I filed https://github.com/census-instrumentation/opencensus-specs/issues/147 to discuss.

Reply:
We are not sure if user should change the defaults right now. We shouldn't export this constant.